testui/elements/testui_element.py

```diff
@@ -156,22 +157,26 @@ def __find_by_collection(self) -> List[WebElement]:
         else:
             raise ElementException(f"locator not supported: {self.locator_type}")
 
-    def is_visible(self, log=True):
+    def is_visible(self, log=True) -> bool:
         is_not = self.__is_not
         try:
-            self.get_element()
-            if log:
+            is_visible = self.get_element().is_displayed()
+
+            if log and is_visible:
                 self.__put_log(f'{self.device_name}: element "{self.locator_type}: {self.locator}" is visible')
```

Comment:
I think that's outside of `is_visible()` scope. Such log would be make sense for `is_present()`(which has to be created at some point)

Reply:
i would argue that the command would produce quite inconsistent logging then. in case element is not present, we do log a message. i would consider that a very important debugging information to know that an element is present but not visible.

Reply:
Oh, didn't understand correctly at first. Yeah, the `try` part needs additional log if `is_visible == False`